Knossos.NET/Views/ModListView.axaml

```diff
@@ -28,11 +28,15 @@
 							</Flyout>
 						</Button.Flyout>
 					</Button>
-					<TextBox Text="{Binding Search}" Width="300"></TextBox>
+					<TextBox Text="{Binding Search}" FontSize="16" Width="300"></TextBox>
 				</WrapPanel>
 				<WrapPanel Margin="10" Grid.Row="0" Grid.Column="1" HorizontalAlignment="Right">
-					<Button Margin="0,0,5,0" Background="Black" FontWeight="Bold" Content="?" ToolTip.Tip="View Color Codes" HorizontalAlignment="Right">
-						<Button.Flyout>
```

Comment:
Might suggest making this a bigger font size rather than using a whole bitmap

Reply:
Though I reckon you added that bitmap so it would be about the same size as the screenshots button?

Reply:
I changed the size at first, but what ended up happening is that it kept aligning to the top, so that the size was the same, but it looked weird because the? was at the top and the screenshots icon was in the middle. The? I called dairy quick job so I can try again to make a better version.

Reply:
And previously, when I have tried to change the font size, I couldn't align the text to the bottom as well.

Reply:
Makes sense!